src/util.ts

```diff
@@ -297,7 +297,8 @@ export function accessPathWithDatum(path: string, datum = 'datum') {
 }
 
 /**
- * Return access with datum to the falttened field.
+ * Return access with datum to the flattened field.
```

Comment:
FYI, the `datum` variable down there is a bit misleading as it can be parent too.  (It confuses me for a bit)

Reply:
Fixed